packages/pretty-format/package.json

```diff
@@ -17,6 +17,7 @@
     "ansi-styles": "^3.2.0"
   },
   "devDependencies": {
+    "@jest/types": "^24.1.0",
```

Comment:
should be in `dependencies` (the interface uses `Plugin`). Then you can remove the eslint ignore comment as well